packages/phone-number-privacy/combiner/src/server.ts

```diff
@@ -0,0 +1,199 @@
+import { ContractKit } from '@celo/contractkit'
+import {
+  CombinerEndpoint,
+  Endpoint,
+  ErrorMessage,
+  getContractKit,
+  loggerMiddleware,
+  rootLogger,
+} from '@celo/phone-number-privacy-common'
+import Logger from 'bunyan'
+import express, { Request, Response } from 'express'
+import { performance, PerformanceObserver } from 'perf_hooks'
+import { CombinerConfig } from '.'
+import { Controller } from './common/controller'
+import { DomainDisableAction } from './domain/endpoints/disable/action'
+import { DomainDisableIO } from './domain/endpoints/disable/io'
+import { DomainQuotaAction } from './domain/endpoints/quota/action'
+import { DomainQuotaIO } from './domain/endpoints/quota/io'
+import { DomainSignAction } from './domain/endpoints/sign/action'
+import { DomainSignIO } from './domain/endpoints/sign/io'
+import { DomainThresholdStateService } from './domain/services/threshold-state'
+import { PnpQuotaAction } from './pnp/endpoints/quota/action'
+import { PnpQuotaIO } from './pnp/endpoints/quota/io'
+import { PnpSignAction } from './pnp/endpoints/sign/action'
+import { PnpSignIO } from './pnp/endpoints/sign/io'
+import { LegacyPnpSignIO } from './pnp/endpoints/sign/io.legacy'
+import { PnpThresholdStateService } from './pnp/services/threshold-state'
+
+require('events').EventEmitter.defaultMaxListeners = 15
+
+export function startCombiner(config: CombinerConfig, kit?: ContractKit) {
+  const logger = rootLogger(config.serviceName)
+
+  logger.info('Creating combiner express server')
+  const app = express()
+  // TODO(2.0.0, logging) get logger to show accurate serviceName
+  // (https://github.com/celo-org/celo-monorepo/issues/9809)
+  app.use(express.json({ limit: '0.2mb' }), loggerMiddleware(config.serviceName))
+
+  // enable cross origin resource sharing from any domain so odis can be interacted with from web apps
+  app.use((_, res, next) => {
+    res.header('Access-Control-Allow-Origin', '*')
+    res.header('Access-Control-Allow-Methods', 'DELETE, PUT, GET, POST, OPTIONS')
```

Comment:
```suggestion
    res.header('Access-Control-Allow-Methods', 'GET, POST, OPTIONS')
```

missed on https://github.com/celo-org/celo-monorepo/pull/9968#discussion_r1001988137